static/app/utils/discover/eventView.tsx

```diff
@@ -528,6 +527,14 @@ class EventView {
       }
     }
 
+    // compare yAxis selections
+    // undefined yAxis values default to count()
+    const currentYAxisValue = this.yAxis ?? 'count()';
+    const otherYAxisValue = other.yAxis ?? 'count()';
+    if (!isEqual(currentYAxisValue, otherYAxisValue)) {
+      return false;
+    }
```

Comment:
Can you add a unit test for this in `tests/js/spec/utils/discover/eventView.spec.jsx`?

Reply:
Good call, test added!